metricbeat/module/elasticsearch/testing.go

```diff
@@ -0,0 +1,29 @@
+package elasticsearch
+
+import "os"
+
+func GetEnvHost() string {
+	host := os.Getenv("ELASTICSEARCH_HOST")
+
+	if len(host) == 0 {
+		host = "127.0.0.1"
+	}
+	return host
+}
+
+func GetEnvPort() string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function GetEnvPort should have comment or be unexported